src/open_inwoner/accounts/tests/test_profile_views.py

```diff
@@ -674,6 +676,39 @@ def test_modify_email_updates_klant_api_but_skip_unchanged_phone(self, m):
         )
 
 
+class TestForm(ErrorMessageMixin, forms.Form):
+    name = forms.CharField(required=True, label="Naam")
+    email = forms.EmailField(required=True, label="E-mailadres")
+
+
+class ErrorMessageMixinTests(TestCase):
```

Comment:
Let's stick to singular nouns for classes. There are many tests in `ErrorMessageMixinTest`, but there is only one class.  I know our code is not consistent, but we should try.